# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1441398418 14400
#      Fri Sep 04 16:26:58 2015 -0400
# Node ID 54eec9c180cb8927785602d6edb5fa0deefca1da
# Parent  fb6fe9b45c41b420e8b8b02b6c9965430e5808ff
* mex.h: Don't typdef bool if it is already defined.

diff --git a/libinterp/corefcn/mex.h b/libinterp/corefcn/mex.h
--- a/libinterp/corefcn/mex.h
+++ b/libinterp/corefcn/mex.h
@@ -47,17 +47,17 @@ SUCH DAMAGE.
 
 #if ! defined (MEX_H)
 #define MEX_H
 
 #define HAVE_OCTAVE
 
 typedef void mxArray;
 
-#if ! defined (__cplusplus)
+#if ! defined (__cplusplus) && ! defined (bool)
 typedef int bool;
 #endif
 
 /* -V4 stuff */
 #if defined (V4)
 #define Matrix mxArray
 #define REAL mxREAL
 #endif
